donkeycar/templates/train.py

```diff
@@ -4,136 +4,117 @@
 Basic usage should feel familiar: python train_v2.py --model models/mypilot
 
 Usage:
-    train.py [--tubs=tubs] (--model=<model>) [--type=(linear|inferred|tensorrt_linear|tflite_linear)]
+    train.py [--tubs=tubs] (--model=<model>) [--type=(linear|inferred|tensorrt_linear|tflite_linear|tflite_r18_lin|tflite_r18_cat)]
 
 Options:
     -h --help              Show this screen.
 """
 
```

Comment:
You don't have the HEAD revision of the `dev` branch, this file is now only a small script supported for backward compatibility.

Reply:
Ok I'll double check then how you train within the HEAD revision.